internal/literals/asthelpers.go

```diff
@@ -0,0 +1,77 @@
+package literals
+
+import (
+	"go/ast"
+	"go/token"
+)
+
+func ident(name string) *ast.Ident {
+	return &ast.Ident{Name: name}
+}
+func intLiteral(value string) *ast.BasicLit {
+	return &ast.BasicLit{
+		Kind:  token.INT,
+		Value: value,
+	}
+}
+
+// name[index]
+func indexExpr(name string, index ast.Expr) *ast.IndexExpr {
+	return &ast.IndexExpr{
+		X:     ident(name),
+		Index: index,
+	}
+}
+
+// convertType(arg)
+func convertExpr(convertType string, arg ast.Expr) *ast.CallExpr {
```

Comment:
instead of `convertType("foo", expr)`, why not `callExpr(ident("foo"), expr)`?

Reply:
Added the callExpr function later, will change.